setup.py

```diff
@@ -1,12 +1,14 @@
 from setuptools import setup, find_packages
 import os
 
-version = '[1;2C.dev0'
+version = '2.7.12'
 maintainer = 'Jonas Baumann'
 
 tests_require = [
     'collective.testcaselayer',
```

Comment:
I think we can remove this now, as well as `Products.PloneTestCase`.

Reply:
Ok, I have removed them.